packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.controller.js

```diff
@@ -0,0 +1,59 @@
+import get from 'lodash/get';
+
+import { DELETE_CONFIRMATION_INPUT } from './scale.constants';
+
+export default class KubernetesNodePoolsScaleCtrl {
+  /* @ngInject */
+  constructor($translate, Kubernetes) {
+    this.$translate = $translate;
+    this.Kubernetes = Kubernetes;
+    this.DELETE_CONFIRMATION_INPUT = DELETE_CONFIRMATION_INPUT;
+  }
+
+  $onInit() {
+    const { autoscale } = this.nodePool;
+    const { lowest, desired, highest } = this.autoscaling.nodes;
+
+    this.isScaling = false;
+    this.autoscaling.autoscale = autoscale;
+    lowest.value = this.nodePool.minNodes;
+    desired.value = this.nodePool.desiredNodes;
+    highest.value = this.nodePool.maxNodes;
```

Comment:
Why not directly declare `lowest/desired/highest` to `this.nodePool` corresponding values ?

Reply:
It's a generic model, to be reusable on some other kube components.